components/Fish.js

```diff
@@ -40,8 +47,7 @@ class Fish extends Component {
           <Image
             style={styles.fishImage}
             source={{
-              uri:
-                "https://images.unsplash.com/photo-1594088979895-6086bd8d1f83?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80",
+              uri: `data:image/gif;base64,${this.props.fish.attributes.image}`,
```

Comment:
This is a good change to make the fish render the image of the fish caught, instead of a default photo.  This will allow the user to see exactly what they had caught to avoid any confusion when cataloging their fish entries.

Reply:
Yes, and thank you so much for researching how to render the base64 uri properly!